sphinx/themes/agogo/static/rtl.css.jinja

```diff
@@ -0,0 +1,156 @@
+{% if theme_is_rtl|tobool %}
+
+/* Core RTL overrides for Agogo theme */
+body,
+div.header div.rel {
+    direction: rtl !important;
+}
+
+body {
+    text-align: right !important;
+}
```

Comment:
There is no need to set `text-align` like this for RTL support. If the HTML document’s `dir` is set to `rtl`, then all text that isn’t forced to a particular alignment will switch to `right` by default.